ui/graphql/client.ts

```diff
@@ -328,12 +338,24 @@ export const client = (
                     variables: { collectionId, offset: 0, limit: 1000 },
                   },
                   (data: any) => {
+
+                    const existingEmails = data.approvedMembersPage?.approvedMembers
+                                              ?.map(member => member.email) || [];
+                    const newInvitedMembers = result.inviteCollectionMembers
+                                              ?.filter(member => (
+                                                existingEmails.indexOf(member.email) === -1
+                                              ));
+
+                    if (newInvitedMembers.length === 0) {
+                      return;
+                    }
+                                                
```

Comment:
What is this code trying to do? Having a hard time to follow, either we add a comment, or we could also invalidate the cache instead to simplify this.

Reply:
https://user-images.githubusercontent.com/38759611/152821570-ebda3b94-ed98-4d80-9a53-e96a95abcaf5.mp4

There was an issue. When you invite the members, the invited members array was concatenated with the existing members without checking whether the new invited members are in the list of existing members or not.

In these lines, I check for existing members and don't push new invited member if it is already there.